src/components/DBEntryManagerPanel/AddOrEditDBEntry/index.tsx

```diff
@@ -0,0 +1,614 @@
+                          }}
+                          ariaLabel={choice.title}
+                        />
+                      );
+                    })
+                  }
+                </ButtonInputGroup>
+              </div>
+            </div>
+          );
+        }
+        return (
+          <div
+            key={field.objectKey}
+            className="mb-2"
+          >
+            <div className="input-group">
+              <span
+                className="AddDBEntry-input-label input-group-text"
+                id="AddDBEntry-form-name-label"
+              >
+                {field.label}
+              </span>
+              <div className="flex-grow-1">
+                <CreatableMultiselect
+                  disabled={disabled}
+                  type={DBEntryFieldType.StringArray}
+                  values={entry[field.objectKey] || []}
+                  onChange={(values) => {
+                    entry[field.objectKey] = values;
+                    dispatch({
+                      type: ActionType.UpdateDBEntry,
+                      DBEntry: entry,
+                    });
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+
+        );
+      }
+      if (field.type === DBEntryFieldType.NumberArray) {
+        return (
+          <div
+            key={field.objectKey}
+            className="mb-2"
+          >
+            <div className="input-group">
+              <span
+                className="AddDBEntry-input-label input-group-text"
+                id="AddDBEntry-form-name-label"
+              >
+                {field.label}
+              </span>
+              <div className="flex-grow-1">
+                <CreatableMultiselect
+                  disabled={disabled}
+                  type={DBEntryFieldType.NumberArray}
+                  values={entry[field.objectKey] || []}
+                  onChange={(values) => {
+                    entry[field.objectKey] = values;
+                    dispatch({
+                      type: ActionType.UpdateDBEntry,
+                      DBEntry: entry,
+                    });
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+
+        );
+      }
+      if (field.type == DBEntryFieldType.Object) {
+        return (
+          <div
+            key={field.objectKey}
+            className="mb-2"
+          >
+            <div className="input-group">
+              <span
+                className="AddDBEntry-input-label input-group-text"
+                id="AddDBEntry-form-name-label"
+              >
+                {field.label}
+              </span>
+              {
+                field.subfields.map((subfield: DBEntryField) => {
+                  return renderEntryField(subfield, disabled);
+                })
+              }
+            </div>
+          </div>
+
+        );
+
+      }
+
+      return null;
```

Comment:
Could you add a comment here explaining that this case shouldn't happen? Or is it expected that this will happen?